src/components/results/shortcircuit/shortcircuit-analysis-result.tsx

```diff
@@ -205,6 +205,10 @@ export const ShortCircuitAnalysisResult: FunctionComponent<
     ]);
 
     useEffect(() => {
+        if (analysisStatus !== RunningStatus.SUCCEED) {
```

Comment:
use Promise.all here as well ?
to avoid code duplication for catch clause for example ?

Reply:
beurk we don't have a `loading` state here ?

Reply:
fixed

you're right, there isn't any loading behaviour for this for now